src/linux/Packaging.Linux/build.sh

```diff
@@ -235,6 +235,12 @@ if [ ! -f "$LINK_TO/git-credential-manager" ]; then
         "$LINK_TO/git-credential-manager" || exit 1
 fi
 
+# Create legacy symlink with older name
+if [ ! -f "$LINK_TO/git-credential-manager-core" ]; then
+    ln -s -r "$INSTALL_TO/git-credential-manager" \
+        "$LINK_TO/git-credential-manager-core" || exit 1
+fi
+
```

Comment:
Do we have any corresponding "uninstall" script that needs an equivalent change?

Reply:
We don't provide an uninstall script for Linux